examples/nextjs-flow/app/hooks/partials.ts

```diff
@@ -0,0 +1,83 @@
+import type {
+  AddPartialTranscriptMessage,
+  AddTranscriptMessage,
+  RecognitionResultAttachesToEnum,
+} from '@speechmatics/flow-client-react';
+
+const CHANGE = 'change';
+
+export class Partials extends EventTarget {
```

Comment:
I've been wondering for a while whether this whole "smart partials" approach is the ideal one. I implemented it this way in the mobile app mainly to get something working, but it might be worth discussing other options.

If we do decide it's the best way though, maybe we can move this into the SDK in some form

Reply:
Yeah let's have a proper chat about it :) 

It would be good to have a super simple example of how they can display them as well maybe?